srt_test.go

```diff
@@ -46,3 +46,9 @@ func TestSRTMissingSequence(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, string(c), w.String())
 }
+
+func BenchmarkOpenSRT(b *testing.B) {
```

Comment:
Could you remove this benchmark and instead add a test to validate reading non-utf8 `srt` returns an error?